spec.md

```diff
@@ -226,11 +233,20 @@ Policies must specify the following fields:
 * v: Version (plain-text, required). Currently only "STS1" is supported.
 * to:  TLS-Only (plain-text, required). If “true,” the receiving MTA requests
   that messages be delivered only if they conform to the STS policy.
+* sto: Sends TLS-Only (plain-text, optional). If "true", the sender's domain
```

Comment:
I realize now that in the existing text I wasn't very careful with distinguishing what the role of the domain is. I.e., for domain example.com who has some policy, in the "to=true" description I refer to this as the "receiving MTA", but in the new " sto=true" description this would be the "receiving MTA." 

I am thinking we should instead define the term "policyholder's domain" or "policy domain' to mean the domain for which the policy is defined, and then we can more clearly say that 

to: If "true", mail to the policy domain SHOULD be delivered only if conformant to the STS policy.
sto: If "true", mail from the policy domain SHOULD be received only if...etc

I will merge your request into my local branch and make some edits and send you a pull request with this change. Sound OK?